sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -576,15 +605,24 @@ static const pi_uint32 ZeCommandListBatchSize = [] {
   return BatchSizeVal;
 }();
 
+bool _pi_queue::getZeCommandListIsCopyList(
+    ze_command_list_handle_t ZeCommandList) {
+  auto it = this->ZeCommandListFenceMap.find(ZeCommandList);
+  if (it == this->ZeCommandListFenceMap.end()) {
+    die("Missing command-list fence map entry");
+  }
+  return it->second.IsCopyCommandList;
+}
+
 // Retrieve an available command list to be used in a PI call
 // Caller must hold a lock on the Queue passed in.
 pi_result _pi_context::getAvailableCommandList(
     pi_queue Queue, ze_command_list_handle_t *ZeCommandList,
-    ze_fence_handle_t *ZeFence, bool AllowBatching) {
+    ze_fence_handle_t *ZeFence, bool IsCopyCommand, bool AllowBatching) {
```

Comment:
name PreferCopyCommandList

Reply:
I used IsCopyCommand because this boolean value specifies if incoming command is a copy command or not. Hope that sounds reasonable.

Reply:
Upon second thoughts..I agree with name change. Thanks

Reply:
Done. Thanks